### PR TITLE
[16.0][MIG] base_sparse_field_list_support

### PR DESCRIPTION
**Pull request by glitchov**
Migrated unchanged

### CODE REVIEW

File: base_sparse_field_list_support/models/fields.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import json
+
+from odoo import fields
```
Comment (lmignon) [inline base_sparse_field_list_support/models/fields.py]:
```suggestion
from odoo.addons.base_sparse_field.models import fields
```

**Review by lmignon — APPROVED** [review]
LGTM, A little change to be more precise about the origin of the field definition...

**Review by lmignon — APPROVED** [review]

**Review by CasVissers-360ERP — APPROVED** [review]

**Review by sbejaoui — APPROVED** [review]

**Review by CasVissers-360ERP — APPROVED** [review]

**Pull request merged**